src/controller/Controller.java

```diff
@@ -1,29 +0,0 @@
-package controller;
-import model.LabyrinthMatrix;
-import view.LabyrinthText;
-
-public class Controller {
-	public static void main(String[] args) {
-		LabyrinthMatrix l = new LabyrinthMatrix();
-		l.setupLabyrinth(-1, 2, 0, 1);
```

Comment:
[informative, maybe even subjective, not mandatory to be taken into consideration]
Have a look on the RAII concept, Resource Acquisition is Initialization. The whole idea is that when a resource is created, it should be ready to use. This helps against resources being used without having been initialized.
To that extent... why not initialize the matrix giving the name of the file from which it would read its data? Something like 
LabyrinthMatrix l = new LabyrinthMatrix("input.txt");
And the details given for "setupLab" are tightly related to what's in the file. Why not read them also from the file, together with the nr. of cols/lines and content? I believe that's actually how a file for the Amazon challenge looked like :). (e.g. nr_cols nr_lines robot_character obstacle_character etc.)